{"level":"ERROR","location":"[On_message:1431] data_ws","message":"name 'Live_data' is not defined","timestamp":"2023-12-31 18:02:37,707+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Python\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1418, in On_message\n    self.OnMessage(message)\n  File \"C:\\Users\\jaypa\\AppData\\Local\\Temp\\ipykernel_13376\\705592368.py\", line 4, in onmessage\n    Live_data(message)\n    ^^^^^^^^^\nNameError: name 'Live_data' is not defined","exception_name":"NameError"}
{"level":"ERROR","location":"[On_message:1431] data_ws","message":"name 'Live_data' is not defined","timestamp":"2023-12-31 18:02:37,759+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Python\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1418, in On_message\n    self.OnMessage(message)\n  File \"C:\\Users\\jaypa\\AppData\\Local\\Temp\\ipykernel_13376\\705592368.py\", line 4, in onmessage\n    Live_data(message)\n    ^^^^^^^^^\nNameError: name 'Live_data' is not defined","exception_name":"NameError"}
{"level":"ERROR","location":"[on_open:1503] data_ws","message":"'FyersModel' object has no attribute 'subscribe'","timestamp":"2023-12-31 18:02:39,465+0530","service":"FyersDataSocket","exception":"Traceback (most recent call last):\n  File \"C:\\Python\\Lib\\site-packages\\fyers_apiv3\\FyersWebsocket\\data_ws.py\", line 1501, in on_open\n    self.OnOpen()\n  File \"C:\\Users\\jaypa\\AppData\\Local\\Temp\\ipykernel_13376\\705592368.py\", line 16, in onopen\n    fyers.subscribe(symbols=symbols, data_type=data_type)\n    ^^^^^^^^^^^^^^^\nAttributeError: 'FyersModel' object has no attribute 'subscribe'","exception_name":"AttributeError"}
